open_humans/models.py

```diff
@@ -88,10 +87,6 @@ def get_queryset(self):
                 .select_related('member')
                 .select_related('member__public_data_participant'))
 
-    def get_by_natural_key(self, username):
```

Comment:
My understanding was that this in place to enable the case-insensitive username and email address lookup. Notably, upon login -- see how it's used in the Django `authenticate` backend here: https://github.com/django/django/blob/stable/2.1.x/django/contrib/auth/backends.py#L16

Removing this seems to be harmless now. The `ACCOUNT_AUTHENTICATION_METHOD` actually becomes relevant if this override is removed, and is resulting in the same behavior.

Sadly I think this might all indicate the custom `User` model is now unnecessary.

I don't think removing the custom User is in scope for this update (and it might be annoying and not worth it). But you should update all the docstrings to indicate what happened. :sweat: Don't just remove the claim that the custom models are enabling this look-up, explain that the model historically had this feature that's no longer present (i.e. the model might exist at this point for an obsolete reason).